Layout_style_v1.css

```diff
@@ -43,8 +45,8 @@ p {
   grid-column-start: 1;
   grid-column-end: span 2;
   padding-bottom: 10px;
-  height: 700px;
-  width: 2000px;
+  width: 1990px;

```

Comment:
I changed the width of the grid-container from 2000 to 1990px, so all of the web structure are all lined up like the navigation bar, header, content, and footer.